scrubber/scrubUtils.ts

```diff
@@ -148,9 +146,20 @@ export function scrubDir(
 
 export function removeFile(filePath: string): Promise<void> {
   const stat = fs.statSync(filePath);
-  if (stat.isDirectory() || stat.isFile()) {
+  if (stat.isDirectory()) {
     return new Promise((resolve, reject) =>
-      fs.rm(filePath, { recursive: true }, (err) => {
+      fs.rmdir(filePath, { recursive: true }, (err) => {
```

Comment:
Just curious, what's the benefit of using directory and file specific functions instead of just rm?

Reply:
There was a node versioning issue, I think the rm command is only supported on the latest node version so I just opted for these instead.